README.md

```diff
@@ -84,6 +84,55 @@ This will add some PPAs to ensure that you can install the required
 golang and mongodb-server versions for precise onwards, in addition to the
 other dependencies.
 
+The recommended way to get dependencies is using godeps, see Dependency management section.
```

Comment:
It's not clear what this has to do with the above. Can you please clarify a little bit further, e.g.:

make install-dependencies" will install *runtime* dependencies. To build Juju, you must also ensure you have the appropriate *build* dependencies. To get the build dependencies, ...